### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_RunningTab_WAT: Color Contrast is less than 4.5:1 for Text/Controls(Terminal, Notebook, Python, ShutDown).

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3999]
**Environment Details:**
Application Name: AzureNotebooks_Jupyter
Login URL: notebooks.azure.com
Browser Details:
Edge: Microsoft Edge 42.17134.1.0
Chrome: Version 68.0.3440.106
Operating System:Windows10

**Repro Steps:**
1.Login with the appropriate credentials to azure notebook account.
2. Go to Get started à libraries à Select any of the libraries à click on clone to clone the library
3. Click on Run or right click and select Run to run the notebook.
4. Navigate to RunningTab and Hit enter.
5. Open WAT Tool.
6. Open Color Contrast Analyser.
7. Verify Color Contrast for Text/Controls(Terminal, Notebook, Python, ShutDown).

**Actual:**
 Color Contrast is less than 4.5:1 for Text/Controls(Terminal, Notebook, Python, ShutDown).

**Expected:**
Color Contrast should be >= 4.5:1 for Text/Controls(Terminal, Notebook, Python, ShutDown).

**User Impact:**
If ColorContrast is <4.5:1 than Low Vision Users face difficulty to See the Text\Controls.

**Attachment:**
![a11y_azurenotebooks_jupyter_runningtab_wat](https://user-images.githubusercontent.com/39204909/45814373-06164e00-bcf3-11e8-9e87-a4a30446a712.png)

**Comment by takluyver** [comment]
Fixed by #4679
